### ISSUE TITLE
Feature Request: Execute OCR via Shortcuts app

### ISSUE THREAD

**Issue by svenwiegand** [holtwick/pdfify#79]
Since macOS Monterey we have the shortcuts app on macOS to simply automate tasks. For automatic batch processing it would be great if pdfify could be used inside shortcut workflows. I still haven't found a PDF OCR tool, which provides this feature.

The optimal solution would be, if pdfify could provide a shortcut action which accepts an input file and produces a searchable PDF as output. But it might be simpler to make pdfify'w new quick action available via shortcuts.

**Comment by holtwick** [comment]
You are right, that would be a very useful. I've put it on the agenda. Thanks!

**Comment by svenwiegand** [comment]
Hi @holtwick, any updates here. At least a command line interface would be a workaround. Currently, there seems to be no option to trigger PDF OCR via any automation.
